mne/decoding/__init__.py

```diff
@@ -10,4 +10,5 @@
 from .time_gen import GeneralizationAcrossTime, TimeDecoding
 from .time_frequency import TimeFrequency
 from .receptive_field import ReceptiveField
+from .time_delaying_ridge import TimeDelayingRidge
```

Comment:
sounds good - so long as you think its fine renaming etc. I'm still a little annoyed that we have a top-level module called `decoding` that does a ton of non-decoding stuff :P